src/std/stxutil.ss

```diff
@@ -13,3 +13,24 @@
 (def (format-id ctx fmt . args)
   (datum->syntax ctx (string->symbol (apply format fmt (map stx-e args)))
                  (stx-source ctx)))
+
+
+;;; Allowing for keywords in macros
+(def (stx-separate-keyword-arguments args (positionals-only? #f)) ;; see std/misc/list#separate-keyword-arguments
```

Comment:
is this used anywhere? I am very dubious about its utility.

Reply:
so if it is not used anywhere, let's remove it and all the associated functions.